elements/xml_form_elements.module

```diff
@@ -14,7 +14,7 @@ define('XML_FORM_ELEMENTS_TABPANEL_THEME', 'tabpanel');
 define('XML_FORM_ELEMENTS_TAGS_THEME', 'tags');
 define('XML_FORM_ELEMENTS_TAGS_CONTENT_THEME', 'tags_content');
 define('XML_FORM_ELEMENTS_TAG_THEME', 'tag');
-define('XML_FORM_ELEMENTS_DATEPICKER_THEME', 'datepicker');
+define('XML_FORM_ELEMENTS_DATEPICKER_THEME', 'datepicker_xml_form_elements');
```

Comment:
Would it make more sense to prefix the element name with the module, instead of as a suffix?